web/cypress/e2e/logging/logs-adminConsole-AggregatedLogs-admin.cy.ts

```diff
@@ -0,0 +1,121 @@
+import { TestIds } from '../../../src/test-ids';
+import { commonTests } from './logs-common-test.cy.ts';
+import { APP_NAMESPACE1,APP_NAMESPACE2,APP_MESSAGE } from './logs-common-test.cy.ts';
+
+describe('Admin in AdminConsole AggregatedLogs', () => {
+  before( function() {
+    cy.uiLoginAsClusterAdmin("first_user");
+    cy.switchToAdmConsole();
+    cy.get('button[data-quickstart-id="qs-nav-workloads"]').click();
+  });
+
+  beforeEach( function() {
+    //Hover on WorkLoad -> Pods page
+    cy.get('[id="page-sidebar"]')
+      .within(() => {
+        cy.contains('a[data-test="nav"]', 'Pods').click();
+      })
+   });
+
+  after( function() {
+    //cy.uiLogoutClusterAdmin("first_user");
+  });
+
+  it('Log Panel top elements', {tags:['@smoke','@aggr']} , () => {
+    //load Aggregated Logs for the first pod in APP_NAMESPACE1
+    cy.get('[data-test-id="namespace-bar-dropdown"]')
```

Comment:
repeated steps like this we can make them reusable in a "utils.ts" file as a "Page Object" concept where you can pass namespace as parameter and do not hardcode it